shard.lock

```diff
@@ -305,6 +305,10 @@ shards:
     git: https://github.com/spider-gazelle/stomp.git
     version: 1.0.1
 
+  stripetease:
+    git: https://github.com/grkek/stripetease.git
+    version: 0.1.0+git.commit.f7693cdda26bad4c58250419b62d1ac7662c8fa7
```

Comment:
What was the reason for building a stripe lib when [this library](https://github.com/confact/stripe.cr) exists?

Reply:
It is lacking the functions we need from what I understand

Reply:
But could you not make a fork and add the functions? I'm into writing a library I just don't know if it's necessary here

Reply:
I have tried the shard and it has some issues, sometimes it crashes on certain values passed to the function, and seems like it hasn't been tested on the never versions of Stripe.